src/scicookie/{{cookiecutter.project_slug}}/docs/index.md

```diff
@@ -11,25 +11,192 @@
 {%- endif %}
 
 ## Features
-{% if cookiecutter.use_bandit == "yes" %}
-- The security of our code: Bandit is a powerful tool that we use in our Python
+
+{% if cookiecutter.project_layout == "src" %}
```

Comment:
same here .. features are the features of the project, not the libraries used by the project.

Reply:
@xmnlab Perfect, did you check the readme.md and the index.md of the project to see if it is ok?